third_party/2and3/termcolor.pyi

```diff
@@ -0,0 +1,21 @@
+# Stub for termcolor: https://pypi.python.org/pypi/termcolor
+from typing import Any, AnyStr, Iterable, Optional
+
+
+def colored(
+    text: AnyStr,
+    color: Optional[str] = ...,
+    on_color: Optional[str] = ...,
+    attrs: Optional[Iterable[str]] = ...,
+) -> str:
+    ...
+
+
+def cprint(
+    text: AnyStr,
+    color: Optional[str] = ...,
+    on_color: Optional[str] = ...,
+    attrs: Optional[Iterable[str]] = ...,
+    **kwargs: Any,
```

Comment:
The `: Any` is redundant, since that's the default.

Reply:
I think it's better to be explicit though. Eventually all arguments in typeshed should be typed.

Reply:
I'm also slightly more for the explicit variant here. @matthiaskramm, let me know if you prefer me to change it.

Reply:
The 'Any' is fine, too.